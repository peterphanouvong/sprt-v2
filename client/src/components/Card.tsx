import { Box } from "@chakra-ui/react";
import React from "react";

interface Props {}

const Card: React.FC<Props> = ({ children }) => {
  return (
    <Box
      shadow="sm"
      borderWidth="1px"
      borderRadius="lg"
      overflow="hidden"
      padding={4}
    >
      {children}
    </Box>
  );
};

export { Card };
