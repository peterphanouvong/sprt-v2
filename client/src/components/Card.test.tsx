import { shallow } from "enzyme";
import { Card } from "./Card";

test("Card snapshot", () => {
  expect(shallow(<Card>Hiya</Card>)).toMatchSnapshot();
});
