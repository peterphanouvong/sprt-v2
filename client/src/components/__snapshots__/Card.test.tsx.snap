// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot 1`] = `
<Box
  borderRadius="lg"
  borderWidth="1px"
  overflow="hidden"
  padding={4}
  shadow="sm"
>
  Hiya
</Box>
`;
